packages/gatsby/src/services/initialize.ts

```diff
@@ -40,17 +40,19 @@ if (
   !isCI()
 ) {
   process.env.GATSBY_EXPERIMENTAL_LAZY_DEVJS = `true`
+  process.env.GATSBY_EXPERIMENTAL_LAZY_IMAGES = `true`
   process.env.GATSBY_EXPERIMENTAL_QUERY_ON_DEMAND = `true`
   process.env.GATSBY_EXPERIMENTAL_DEV_SSR = `true`
 
   reporter.info(`
-Three fast dev experiments are enabled, Lazy Bundling, Query on Demand, and Development SSR.
+Four fast dev experiments are enabled, Lazy Bundling, Lazy Images, Query on Demand, and Development SSR.
 
 Please give feedback on their respective umbrella issues!
 
 - https://gatsby.dev/lazy-devjs-umbrella
 - https://gatsby.dev/query-on-demand-feedback
 - https://gatsby.dev/dev-ssr-feedback
+- https://gatsby.dev/lazy-images-feedback
   `)
```

Comment:
This is tricky. We can't guarantee that lazy images are enabled, because user might not use compatible version of `gatsby-plugin-sharp` plugin. It feels like we would want to have Lazy images included in this umbrella flag, but maybe it should just be all handled in `gatsby-plugin-sharp` plugin at the cost of less clear notification?

Or possibly we need to edit text a little "Lazy images (if supported by used gatsby-plugin-sharp version)" (or so?)

Reply:
We actually have plugin versions inside `initialize` call available. So maybe just wrap this warning with a function and call it from the `initialize` (and pass plugins as an option to tweak the message?)

Reply:
It is doable for sure - but it is much more involved.

The worry I also have about delaying setting those env vars is that I don't know if all those experiments will work correctly if setting env var is delayed (i.e. does some of those do something conditional at "import time" vs runtime). I was very careful with query on demand case so it works if it's set after we read gatsby-config but I can't guarantee it's the case for other experiments

Reply:
For now - I would vote to edit the text we have currently to mention that it relies on proper version of gatsby-plugin-sharp and I will do investigation about other experiments and if we can move this entire block to after we read gatsby-config tomorrow

Reply:
We can set env vars immediately but output the message later (when we have data about plugin versions)?

Reply:
I fear we _can_ get into weird timing situations with those (we set variables, then something unset them and then when we want to print message something happend and stuff can get all weird). Not saying it will happen (likely not), but it feels very brittle and errorprone. 

Given that `FAST_DEV` is something that people really need to opt-in, I think it would be acceptible to do minor edit to notification text to handle my concern of not compatible sharp plugin version and delay doing really nice message once we have APIs for experiments/feature flags management ( https://github.com/gatsbyjs/gatsby/pull/28296 / https://github.com/gatsbyjs/gatsby/pull/28269 ) in place to not put too much work into trying to fix those issues with our problematic env var approach as those will be refactored anyway

Reply:
Yup, updated the text 👍